src/box/lua/config/utils/schema.lua

```diff
@@ -719,6 +721,146 @@ end
 
 -- }}} <schema object>:set()
 
+-- {{{ <schema object>:filter()
+
+local filter_impl
+filter_impl = function(schema, data, f, ctx)
```

Comment:
Minor: Why do you need such a complex definition here, but not for `get_impl`/`set_impl`?

Reply:
Thanks for catching it! All the recursive functions were defined this was in my early PoC, then I changed it to just `local function <...>_impl(<...>) <...> end`. But forgot to change this function.

Fixed.